licenses/inlong-agent/notices/NOTICE-jakarta.validation-api.txt

```diff
@@ -1,42 +1,42 @@
-# Notices for Eclipse Jakarta Bean Validation
-
-This content is produced and maintained by the Eclipse Jakarta Bean Validation
-project.
-
-* Project home: https://projects.eclipse.org/projects/ee4j.bean-validation
-
-## Trademarks
-
- Jakarta Bean Validation is a trademark of the Eclipse Foundation.
-
-## Copyright
-
-All content is the property of the respective authors or their employers. For
-more information regarding authorship of content, please consult the listed
-source code repository logs.
-
-## Declared Project Licenses
-
-This program and the accompanying materials are made available under the terms
-of the Apache License, Version 2.0 which is available at
-https://www.apache.org/licenses/LICENSE-2.0.
-
-SPDX-License-Identifier: Apache-2.0
-
-## Source Code
-
-The project maintains the following source code repositories:
-
- * [The specification repository](https://github.com/eclipse-ee4j/beanvalidation-spec)
- * [The API repository](https://github.com/eclipse-ee4j/beanvalidation-api)
- * [The TCK repository](https://github.com/eclipse-ee4j/beanvalidation-tck)
-
-## Third-party Content
-
-This project leverages the following third party content.
-
-Test dependencies:
-
- * [TestNG](https://github.com/cbeust/testng) - Apache License 2.0
- * [JCommander](https://github.com/cbeust/jcommander) - Apache License 2.0
- * [SnakeYAML](https://bitbucket.org/asomov/snakeyaml/src) - Apache License 2.0
+# Notices for Eclipse Jakarta Bean Validation
```

Comment:
This PR has modified many files. Will gitatattributes modify the contents of the license and notice files?

Reply:
it will not affect the content of license and notice files.